File: Python/Komik.py
from Buku import Buku

class Komik(Buku):
    def __init__(self, kodeProduk, namaProduk, harga, penulis, penerbit, tahunTerbit,
                 genre, jumlahHalaman, volume):
        super().__init__(kodeProduk, namaProduk, harga, penulis, penerbit, tahunTerbit)
        self.genre = genre
        self.jumlahHalaman = jumlahHalaman
        self.volume = volume

    def getGenre(self):
        return self.genre

    def getJumlahHalaman(self):
        return self.jumlahHalaman

    def getVolume(self):
        return self.volume

    def setGenre(self, genre):
        self.genre = genre

    def setJumlahHalaman(self, jumlahHalaman):
        self.jumlahHalaman = jumlahHalaman

    def setVolume(self, volume):
        self.volume = volume

File: Java/Komik.java
public class Komik extends Buku {
    private String genre;
    private int jumlahHalaman;
    private int volume;

    public Komik(String kodeProduk, String namaProduk, int harga,
                 String penulis, String penerbit, int tahunTerbit,
                 String genre, int jumlahHalaman, int volume) {
        super(kodeProduk, namaProduk, harga, penulis, penerbit, tahunTerbit);
        this.genre = genre;
        this.jumlahHalaman = jumlahHalaman;
        this.volume = volume;
    }

    public String getGenre() { return genre; }
    public int getJumlahHalaman() { return jumlahHalaman; }
    public int getVolume() { return volume; }

    public void setGenre(String genre) { this.genre = genre; }
    public void setJumlahHalaman(int jumlahHalaman) { this.jumlahHalaman = jumlahHalaman; }
    public void setVolume(int volume) { this.volume = volume; }
}
